### DISCUSSION TITLE
Agenda for the Flatcar Release Team Call on Tuesday, 27th of July 9pm IST / 5:30pm CEST / 3:30pm GMT / 11:30am EDT / 8:30am PST

### DISCUSSION THREAD

**Discussion by sayanchowdhury** [flatcar/Flatcar#811] [Flatcar Developer Sync]
## Agenda for the Flatcar Release Team Call on Tuesday, 27th of Jul 9pm IST / 5:30pm CEST / 3:30pm GMT / 11:30am EDT / 8:30am PST

* Zoom link: [https://us06web.zoom.us/j/82054240491](https://us06web.zoom.us/j/82054240491)
  * Meeting ID: 820 5424 0491
  * Passcode: 444888
- Youtube live stream / recording: https://www.youtube.com/watch?v=P8IjaNuKnxQ
- Community Calls calendar (all future calls): [link](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [link](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)

## Welcome
- Brief intro/check-in of the new participants in the Zoom call. Please introduce yourself and share what brings you here today.

## News
- Talk about the current release and problems faced
- Status of the current set of issues in the Stable and the plan for the fix

## Release planning
- Ongoing and upcoming releases (https://github.com/orgs/flatcar-linux/projects/7/views/8)

## Q&A
- Questions from community participants, answered by the Flatcar maintainers

**Comment by jepio** [comment]
> Status of the current set of issues in the Stable and the plan for the fix

Systemd v250 landed in stable, and several issues have surfaced:
* accidentally enabled systemd stub-resolver.conf, report came in via slack (thanks @whites11). Patch merged to all branches https://github.com/flatcar-linux/coreos-overlay/pull/2057
* behavior of systemd-networkd changed wrt. keeping IP addresses configured on wireguard interfaces https://github.com/flatcar-linux/Flatcar/issues/808
* gcc libs (libgcc/libstdc++) were accidentally placed in the wrong library directory (/usr/lib instead of /usr/lib64), may show up on upgrade to stale dynamic linker cache (/etc/ld.so.cache) https://github.com/flatcar-linux/Flatcar/issues/809. PR open: https://github.com/flatcar-linux/scripts/pull/391
* systemd-networkd failure during boot on certain bare metal platforms and linode. https://groups.google.com/g/flatcar-linux-user/c/44jLwvlogqQ / https://github.com/flatcar-linux/Flatcar/issues/807/. Insufficient logs to root cause at this time, if you're affected reach out so that we can get to a resolution faster.

**Comment by t-lo** [comment]
@sayanchowdhury  There's an umbrella tracking issue for the 3227.2.0 networkd issues: https://github.com/flatcar-linux/Flatcar/issues/810 . @miao0miao is tracking the overall status.
